src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -646,7 +650,8 @@ void mp::QemuVirtualMachine::initialize_vm_process()
             }
             else
             {
-                mpl::log(mpl::Level::error, vm_name, fmt::format("error: {}", process_state.error->message));
+                auto log_level = force_shutdown ? mpl::Level::info : mpl::Level::error;
+                mpl::log(log_level, vm_name, fmt::format("error: {}", process_state.error->message));
```

Comment:
I think we should reset `force_shutdown` value to false after a force shutdown, otherwise the started VM will have `force_shutdown` value true without forcefully shutdown. Then the operations should report the message as an error will have it silenced. (By the way, I am not sure which exact operation should trigger that, be free to share if you know. )

If I did not understand wrong, `Process::error_occurred` happens first, then `Process::finished`, so maybe add `force_shutdown = false;` here which is the end of the sequence of actions.